gamePage.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <script src="script.js" defer></script>
+    <title>Pro-To-Type Typing Speed Game</title>
+</head>
+
+<body>
+    <main>
+        <div class="linkToRules">
+            <a href="index.html">Back to rules</a>
+        </div>
+        <h2>Copy the text below: </h2>
+        <div id="textsToTypeContainer">
+            <div id="resultsPopup">
+                <h2>RESULTS:</h2>
+                <div id="results">
+                    <div id="wordsPerMinuteContainer">
+                        <h3 id="wordsPerMinuteSubheading">Words per minute: </h3>
+                        <p id="wordsPerMinuteResult">Test</p>
+                    </div>
+                    <div id="typingAccuracyContainer">
+                        <h3 id="accuracySubheading">Accuracy: </h3>
+                        <p id="accuracyResult">Test</p>
+                    </div>
+                </div>
+                <p id="scrollDownText">Scroll down to see which words you got right.</p>
+            </div>
+        </div>
+        <div id="inputAndTimerContainer">
+            <label class="hidden_label_text_input" for="textInput">Start typing here</label>
```

Comment:
this is the only class that uses snake_case, change it to camelCase like all the rest